lib/Runtime/Library/IntlEngineInterfaceExtensionObject.cpp

```diff
@@ -1058,24 +1101,12 @@ enum class LocaleDataKind
             return scriptContext->GetLibrary()->GetUndefined();
         }
 
-        Var toReturn = nullptr;
-        JavascriptString *localeStrings = JavascriptString::FromVar(args.Values[1]);
-        PCWSTR passedLocale = localeStrings->GetSz();
-
 #if defined(INTL_ICU)
-        char16 resolvedLocaleName[ULOC_FULLNAME_CAPACITY] = { 0 };
-        if (ResolveLocaleBestFit(passedLocale, resolvedLocaleName))
-        {
-            toReturn = JavascriptString::NewCopySz(resolvedLocaleName, scriptContext);
-        }
-        else
-        {
-#ifdef INTL_ICU_DEBUG
-            Output::Print(_u("Intl::ResolveLocaleBestFit returned false: EntryIntl_ResolveLocaleBestFit returning null to fallback to JS\n"));
-#endif
-            toReturn = scriptContext->GetLibrary()->GetNull();
-        }
+        AssertOrFailFastMsg(false, "Intl-ICU does not implement ResolveLocaleBestFit");
+        return nullptr;
```

Comment:
>nullptr [](start = 15, length = 7)

not sure we should return nullptr into a Var. Maybe we should return scriptContext->GetLibrary()->GetUndefined() instead?

Reply:
Its an AssertOrFailFast, so this is only there to satisfy the compiler

Reply:
Also I tried to axe as much "Uh... idk return undefined" as I could because if we don't know what to do from the native side, chances are we don't know what to do on the JS side either and wouldn't handle undefined gracefully.